rca/enquire_to_study/wagtail_hooks.py

```diff
@@ -1,12 +1,27 @@
+from wagtail.contrib.modeladmin.helpers import PermissionHelper
 from wagtail.contrib.modeladmin.options import ModelAdmin, modeladmin_register
 
-from rca.enquire_to_study.models import Submission
+from rca.enquire_to_study.models import EnquiryFormSubmission
 
 
-class SubmissionAdmin(ModelAdmin):
-    model = Submission
-    menu_label = "Submission"
-    menu_icon = "pick"
+class EnquiryFormSubmissionPermissionHelper(PermissionHelper):
+    def user_can_list(self, user):
+        return True
+
+    def user_can_create(self, user):
+        return False
+
+    def user_can_edit_obj(self, user, obj):
+        return True
+
+    def user_can_delete_obj(self, user, obj):
+        return True
+
+
+class EnquiryFormSubmissionAdmin(ModelAdmin):
+    model = EnquiryFormSubmission
+    menu_label = "Enquiry Submissions"
```

Comment:
'Enquiry Form Submissions ' is too long, therefore I named it Enquiry Submissions instead